applications/nrf_desktop/doc/battery_charger.rst

```diff
@@ -0,0 +1,53 @@
+.. _battery_charger:
+
+Battery charger module
+######################
+
+The ``battery_charger`` module is responsible for battery charging control.
+
+Module Events
+*************
+
++------------------+---------------------+---------------------+-------------------------+------------------+
+| Source Module    | Input Event         | This Module         | Output Event            | Sink Module      |
++==================+=====================+=====================+=========================+==================+
+|                  |                     | ``battery_charger`` | ``battery_state_event`` | :ref:`led_state` |
++------------------+---------------------+                     +-------------------------+------------------+
+| :ref:`usb_state` | ``usb_state_event`` |                     |                         |                  |
++------------------+---------------------+---------------------+-------------------------+------------------+
+
+Configuration
+*************
+
+The module implemented in ``battery_charger.c`` uses the Zephyr :ref:`zephyr:gpio` driver to control and monitor battery charging. For this reason, you should set :option:`CONFIG_GPIO` option.
+
+Set option ``CONFIG_DESKTOP_BATTERY_CHARGER_DISCRETE`` to enable the module (by default ``CONFIG_DESKTOP_BATTERY_CHARGER_NONE`` is selected - module is disabled).
```

Comment:
```suggestion
 By default, the module is disabled and the ``CONFIG_DESKTOP_BATTERY_CHARGER_NONE`` option is selected.
 Set the option ``CONFIG_DESKTOP_BATTERY_CHARGER_DISCRETE`` to enable the module.
```
Don't use en or em dashes as in Polish. They don't have exactly the same meaning.
Is this option set in CONFIG_GPIO?

Reply:
These are two different options:
`CONFIG_GPIO` is Zephyr option to enable GPIO
Other `CONFIG_DESKTOP...` options are our application options used to enable the module.